cmd/minikube/cmd/root.go

```diff
@@ -126,8 +126,7 @@ func setFlagsUsingViper() {
 }
 
 func init() {
-	RootCmd.PersistentFlags().StringP(config.MachineProfile, "p", constants.DefaultMachineName, `The name of the minikube VM being used.  
-	This can be modified to allow for multiple minikube instances to be run independently`)
+	RootCmd.PersistentFlags().StringP(config.MachineProfile, "p", constants.DefaultMachineName, `This can be set to allow having multiple instances of minikube independently`)
```

Comment:
sorry I meant to keep the first line and add the second line 
`The name of the minikube VM being used. This can be set to allow having multiple instances of minikube independently`

Reply:
Oh. Sorry. Fixing

Reply:
fixed